ui/src/main/js/page/user/user/UserCopyRowAction.js

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import { createUseStyles } from 'react-jss';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import UserCopyModal from 'page/user/user/UserCopyModal';
+
+const useStyles = createUseStyles({
+    copyCell: {
+        '&:hover': {
+            cursor: 'pointer',
+        }
+    }
+});
+
+const UserCopyRowAction = ({ data }) => {
+    const classes = useStyles();
+    const [showModal, setShowModal] = useState(false);
+    const [selectedData, setSelectedData] = useState(data);
+    const selectedUsername = data.username;
+    const email = selectedData.emailAddress ? selectedData.emailAddress : '';
+
+    function handleClick() {
+        setShowModal(true);
+        setSelectedData(userData => ({...userData, id: null, username: '', password: '', emailAddress: email }))
```

Comment:
```suggestion
        setSelectedData((userData) => ({...userData, id: null, username: '', password: '', emailAddress: email }))
```
lambda expressions should have parenthesis around the variable.